src/storage/mvcc/reader.rs

```diff
@@ -768,7 +768,8 @@ mod tests {
         fn rollback(&mut self, pk: &[u8], start_ts: u64) {
             let k = make_key(pk);
             let snap = RegionSnapshot::from_raw(Arc::clone(&self.db), self.region.clone());
-            let mut txn = MvccTxn::new(snap, start_ts, None, IsolationLevel::SI, true);
+            let mut txn = MvccTxn::new(snap, start_ts, None, IsolationLevel::SI, true)
```

Comment:
Why not change it into builder style completely?

Reply:
I think it's ok since it is only used for tests. Maybe we could add some comments for function `collaspe_rollback` to indicate it's only used in the test?

Reply:
@AndreMouche 

seems it is strange that you use a builder style directly following the new function.

Reply:
Actually builder style only can apply to MvccReader, not MvccTxn.

Reply:
why can't add a builder for MvccTxn?

Reply:
Most of the arguments belong to MvccReader.